<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hash Check</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="labelQueue">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Queue:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelFile">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>File:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QProgressBar" name="progressBarQueue">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelAlgorithm">
        <property name="text">
         <string>Algorithm: </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>22</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QTreeWidget" name="treeWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="itemsExpandable">
         <bool>false</bool>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>false</bool>
        </property>
        <attribute name="headerMinimumSectionSize">
         <number>50</number>
        </attribute>
        <column>
         <property name="text">
          <string>File</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Algorithm</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Checksum</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Reference</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_File"/>
   <addaction name="actionDel_Entry"/>
   <addaction name="actionClear_List"/>
   <addaction name="separator"/>
   <addaction name="actionResume"/>
   <addaction name="actionPause"/>
   <addaction name="actionSkip"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionImport"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="hash_check.qrc">
     <normaloff>:/menu/import</normaloff>:/menu/import</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="toolTip">
    <string>Import checksum index file</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="hash_check.qrc">
     <normaloff>:/menu/export</normaloff>:/menu/export</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export checksum index file</string>
   </property>
  </action>
  <action name="actionAdd_File">
   <property name="icon">
    <iconset resource="hash_check.qrc">
     <normaloff>:/menu/add</normaloff>:/menu/add</iconset>
   </property>
   <property name="text">
    <string>Add File</string>
   </property>
   <property name="toolTip">
    <string>Add file to hash checking queue</string>
   </property>
  </action>
  <action name="actionDel_Entry">
   <property name="icon">
    <iconset resource="hash_check.qrc">
     <normaloff>:/menu/del</normaloff>:/menu/del</iconset>
   </property>
   <property name="text">
    <string>Del Entry</string>
   </property>
   <property name="toolTip">
    <string>Delete selected entries from the list</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="icon">
    <iconset resource="hash_check.qrc">
     <normaloff>:/menu/play</normaloff>:/menu/play</iconset>
   </property>
   <property name="text">
    <string>Resume</string>
   </property>
   <property name="toolTip">
    <string>Resume hash scanning</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="hash_check.qrc">
     <normaloff>:/menu/pause</normaloff>:/menu/pause</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause hash scanning</string>
   </property>
  </action>
  <action name="actionSkip">
   <property name="icon">
    <iconset resource="hash_check.qrc">
     <normaloff>:/menu/skip</normaloff>:/menu/skip</iconset>
   </property>
   <property name="text">
    <string>Skip</string>
   </property>
   <property name="toolTip">
    <string>Skip current hash scan</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="hash_check.qrc">
     <normaloff>:/menu/stop</normaloff>:/menu/stop</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop hash scanning and clear queue</string>
   </property>
  </action>
  <action name="actionClear_List">
   <property name="icon">
    <iconset resource="hash_check.qrc">
     <normaloff>:/menu/clear</normaloff>:/menu/clear</iconset>
   </property>
   <property name="text">
    <string>Clear List</string>
   </property>
   <property name="toolTip">
    <string>Clear the hash result list</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="hash_check.qrc">
     <normaloff>:/menu/about</normaloff>:/menu/about</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Open the About dialog</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="hash_check.qrc"/>
 </resources>
 <connections/>
</ui>
